src/main/registry.ts

```diff
@@ -53,17 +56,17 @@ export interface IRegistry {
 }
 
 export const SERVER_TOKEN_PREFIX = 'jlab:srvr:';
+const MIN_JLAB_VERSION_REQUIRED = '3.0.0';
 
 export class Registry implements IRegistry, IDisposable {
   constructor() {
-    const minJLabVersionRequired = '3.0.0';
-
     this._requirements = [
       {
         name: 'jupyterlab',
         moduleName: 'jupyterlab',
         commands: ['--version'],
-        versionRange: new semver.Range(`>=${minJLabVersionRequired}`)
+        versionRange: new semver.Range(`>=${MIN_JLAB_VERSION_REQUIRED}`),
+        pipCommand: 'jupyterlab'
```

Comment:
should it include `>=${MIN_JLAB_VERSION_REQUIRED}` as well?

Reply:
it was causing installation to fail on Windows. I will take another look.